cupyx/jit/_builtin_funcs.py

```diff
@@ -15,6 +15,31 @@
 
 class RangeFunc(BuiltinFunc):
 
+    def __call__(self, *args, unroll=None):
+        """Range with loop unrolling support.
+
+        Args:
+            start (int):
+                Same as that of built-in range.
+            stop (int):
+                Same as that of built-in range.
+            step (int):
+                Same as that of built-in range.
+            unroll (int or bool or None):
+                If ``True``, add ``#pragma unroll`` directive before the loop.
+                If ``False``, add ``#pragma unroll(1)`` directive before the
+                loop to disable unrolling.
+                If ``int`` type, add ``#pragma unroll(n)`` directive before the
+                loop, where ``n`` means the number of iterations to unroll.
+                If ``None``, leave the control of loop unrolling to compiler.
+
+        .. seealso:: `#pragma unroll`_
+
+        .. _#pragma unroll:
+            https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#pragma-unroll
+        """
+        super.__call__(self)
+
     def call(self, env, *args, **kwargs):
```

Comment:
My impression is that @asi1024 prefers it to be explicit (https://github.com/cupy/cupy/pull/5335#discussion_r654654471):
```suggestion
    def call(self, env, *args, *, unroll=None):
```

Reply:
`unroll` is already a keyword-only argument, note the `*args` before `unroll` 😄